src/plays/fun-quiz/FrontScreen.scss

```diff
@@ -0,0 +1,174 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap');
```

Comment:
Do you need this font? @Angryman18 

If yes, Can you please make make sure it doesn't override the font of reactplay when switch between this play and others?

Reply:
Yes its removed.